### DISCUSSION TITLE
Question about  loot tables

### DISCUSSION THREAD

**Discussion by Danimal696** [flareteam/flare-game#869] [Q&A]
Hi, I´m making a generator for items that will randomize all items stats, since flare can´t do this and all items needs to be predefined into it´s .txt; I have a question in the way loot drops works. As far as i know each time you kill something you "roll a dice" for each and every item in the stated loot table until you get lucky or fail the last table item and get nothing. So that would mean that a table with 1000+ items is always a sure drop even at a 1% chance for each item?. It´s possible to go lower than 1%?

**Comment by dorkster** [comment]
The dice roll doesn't happen for each item in the loot table. Instead a single dice roll happens each time a piece of loot is set to drop. We then grab all the items that have a chance less than or equal to the roll value. If there are multiple items that meet the criteria, one is picked at random from that pool.

To be guaranteed a drop, there must be one or more items with a chance of 100. There's also the special case of setting chance to 0 (aka "fixed" chance), which will drop the item (and remove it from the loot table) before moving on to the random items. Fixed chance is mostly used for quest items, so don't worry about it for random loot tables.

Loot chances are floats, so you can use values less than 1%.
